__tests__/application/toJSON.test.js

```diff
@@ -1,8 +1,11 @@
 'use strict'
 
+const { describe, it } = require('node:test')
 const assert = require('assert')
 const Koa = require('../..')
 
+process.env.NODE_ENV = 'test'
```

Comment:
Why is `NODE_ENV` required specifically for this test?

Reply:
[changed](https://github.com/koajs/koa/pull/1845/commits/492ba1a39af3f3ec33299cfa872984412b3afecf#diff-bda586e9e7e6feab76eb0c96e72b20cb2b06a6a1b09f7f3da19aa18f6863b7caL7-L11), moved this env to params